addon/components/links-with-follower.js

```diff
@@ -210,21 +210,20 @@ export default Ember.Component.extend({
    * Animates a follower to its final width / position.
    *
    * @param  {Boolean} animate=true If the follower should animate or not
-   * @param  {Object} options={}    Animate options passed to velocity or jQuery
+   * @param  {Object} options={}    Animate options passed to jQuery's css method
```

Comment:
should we reword this since we don't pass all options but only use width and left?